templates/workspace_services/guacamole/user_resources/guacamole-azure-importreview-vm/reset_password.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Same as above: think this is not necessary

Reply:
Ok! I have deleted them both now